src/contracts/core/DelegationManager.sol

```diff
@@ -442,6 +442,60 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         }
     }
 
+    /**
+     * @notice Called by AVSs to register an operator with the AVS.
+     * @param operator The address of the operator to register.
+     * @param signatureWithSaltAndExpiry The signature, salt, and expiry of the operator's signature.
+     */
+    function registerOperatorWithAVS(address operator, ISignatureUtils.SignatureWithSaltAndExpiry memory signatureWithSaltAndExpiry) external {
```

Comment:
Would prefer this is a multi-line declaration, since it's quite long.

Additionally, I'm a big fan of just calling these signature parameters `signature` rather than the super long rewording of the struct name.

Reply:
Or even better, `operatorSignature` because that communicates important info.

Reply:
Talked to yash about this yesterday. Unclear that we need the signature. Can't the contract just check that the tx originator is the operator address?